subsys/bluetooth/mesh/shell/shell_scheduler_cli.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <bluetooth/mesh/models.h>
+#include <shell/shell.h>
+#include <sys/math_extras.h>
+
+#include "mesh/net.h"
+#include "mesh/access.h"
+#include "../model_utils.h"
+
+#include "shell_utils.h"
+
+static struct bt_mesh_model *mod;
+static struct bt_mesh_schedule_entry set_entry;
+
+static int cmd_get(const struct shell *shell, size_t argc, char *argv[])
+{
+	if (!mod && !shell_model_first_get(BT_MESH_MODEL_ID_SCHEDULER_CLI, &mod)) {
+		return -ENODEV;
+	}
+
+	struct bt_mesh_scheduler_cli *cli = mod->user_data;
+	uint16_t rsp;
+
+	int err = bt_mesh_scheduler_cli_get(cli, NULL, &rsp);
+
+	if (!err) {
+		shell_print(shell, "Status: %d", rsp);
+	}
+
+	return err;
+}
+
+static void schedule_print(const struct shell *shell, int err, struct bt_mesh_schedule_entry *rsp)
+{
+	if (!err) {
+		shell_print(shell,
+			    "Day: %02d month: %02d year: %02d,\n"
```

Comment:
Just nagging but it would be more readable if each parameter will be in the new line.